spec/system/candidate_interface/carry_over/candidate_carries_over_different_application_states_spec.rb

```diff
@@ -8,6 +8,26 @@
     @application_form = create(:application_form, :completed, :pre_continuous_applications, candidate: @candidate)
   end
 
+  # To avoid infinite redirects for "QA" data or any other data states that
+  # might cause odd behaviour on carry over
+  scenario 'Candidate sees complete page when submitted at has value but courses choices does not have submission' do
+    given_i_have_an_empty_submitted_application_from_last_cycle
+    and_i_am_signed_in_as_a_candidate
```

Comment:
Why do you need this step?

Reply:
Because the redirect to carry over is after sign in

Reply:
But do you have to sign in, sign out and sign in again?

Reply:
Oh I see we do have two sign ins. I removed the second one.